scripts/apps/authoring/authoring/directives/AuthoringDirective.js

```diff
@@ -410,6 +408,12 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
                 return $rootScope.config.features && $rootScope.config.features.useTansaProofing;
             };
 
+            var cleanTansa = $rootScope.$on('tansa:end', (_, cancel) => {
```

Comment:
Just do:
```js
var deregisterTansa = $rootScope.$on('tansa:end', afterTansa);
```
The returned function on event registration is called "_deregister_" in Angular terms. So let's use that.